packages/Spellcheck-Core.package/SpellcheckCorrectionUI.class/instance/inputText..st

```diff
@@ -1,7 +1,7 @@
-as yet unclassified
+initialize
 inputText: aText
 
-	inputText := aText.
+	inputText := aText asText.	"might be a string"
```

Comment:
In case we want to make this UI usable without a workspace (by using `inputText:` and not `input:`) we need to make sure we are getting a Text and not a string. The method `replaceFrom: To: with:` of a String is only working if we don't change the length.
The problem with `asText` is that we no longer work on the actual text (in the sense of the actual words), but on a copy of the text in the form of a Text-instance.
Thus we change the words of the Text-instance when we are replacing words but the string remains unchanged. 
(it still works if we pass a Text-Instance to `inputText:`)

I am not sure which I like less...

Reply:
mh, I think I disslike the ```replaceFrom: To: with:``` more, since not changing the length seems quite hacky to me, but in the end its up to you of course

Why do the builds fail on 5.0 though?

Reply:
Travis failed to connect to a website which was needed to load some our project. I've restarted it and it works